modules/rank-eval/src/test/java/org/elasticsearch/index/rankeval/RankEvalRequestTests.java

```diff
@@ -35,7 +35,15 @@
 import java.util.Map.Entry;
 import java.util.Set;
 
-@ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.SUITE)
+@ESIntegTestCase.ClusterScope(
+        scope = ESIntegTestCase.Scope.SUITE,
+        minNumDataNodes = 1,
+        numDataNodes = 2,
+        maxNumDataNodes = 3,
+        numClientNodes = 1,
+        transportClientRatio = 0.5,
+        supportsDedicatedMasters = true,
+        randomDynamicTemplates = true)
```

Comment:
Was this added recently in other similar tests? Just wondering if I need to use this in other places in the future as well, thats why I'm curious.